client/src/tests/interface_tests.rs

```diff
@@ -373,3 +373,36 @@ fn test_load_multiple_clients_from_snapshot() {
         assert!(result.is_ok(), "Failed to load client from snapshot path {:?}", result);
     });
 }
+
+#[test]
+fn test_load_client_from_non_existing_snapshot() {
+    let client_path = "my-awesome-client-path";
+    let stronghold = Stronghold::default();
+    let snapshot_path = SnapshotPath::named("idkfa.snapshot");
+    let password = rand::fixed_bytestring(32);
+    let keyprovider = KeyProvider::try_from(password).expect("KeyProvider failed");
+
+    let result = match stronghold.load_client_from_snapshot(client_path, &keyprovider, &snapshot_path) {
+        Err(client_error) => {
+            std::mem::discriminant(&client_error)
+                == std::mem::discriminant(&ClientError::SnapshotFileMissing("obo".to_string()))
+        }
+        Ok(_) => false,
+    };
+
+    assert!(result)
+}
+
+#[test]
+fn test_create_snapshot_file_in_custom_directory() {
+    let client_path = "my-awesome-client-path";
```

Comment:
`client_path` is not used into `test_create_snapshot_file_in_custom_directory` fn